documentation/docs/13-configuration.md

```diff
@@ -27,6 +27,7 @@ module.exports = {
 		},
 		host: null,
 		hostHeader: null,
+		hydrate: true,
```

Comment:
I wonder if we should try to group these a bit like:
```
rendering: {
  ssr: true,
  csr: true,
  hydrate: true
}
```

Reply:
I did wonder about that but I think 'rendering' is the wrong word. Any alternatives?

Reply:
`mode` ?

Reply:
`execution`?

Reply:
`process`? `format`? `rubric`?

Reply:
For what it's worth, I like the original `rendering` or `render` to align with `prerender`.

Reply:
I don't think `render` is ideal because routing !== rendering. `mode` to me feels like it implies mutual exclusivity, when really these options are unrelated (except for the fact that `ssr` and `hydrate` both being false results in a broken app). Which is perhaps an argument for leaving them as top-level items

Reply:
Another option would be just `ssr` and `csr` under `rendering` and then put `hydrate` as a top-level item

Reply:
except that `csr` stands for 'client-side _routing_'. it's very confusing!

Reply:
Oh, no I actually meant client-side rendering - not routing. I was just suggesting that when that was implemented it could go there. Not that we rename `router` to `csr`. I guess I can see how that wasn't clear

So for this PR I'd probably suggest something like:
```
rendering: {
  hydrate: true,
  ssr: true
},
router: true
```

Reply:
Since `hydrate` and `ssr` both default to `true`, and can't both be `false` without making the app useless, there's never a time when you'd specify both. So all we achieve by nesting them is turning

```js
ssr: false
```

into

```js
rendering: {
  ssr: false
}
```

and

```js
hydrate: false
```

into

```js
rendering: {
  hydrate: false
}
```

It's just extra keypresses with no practical value as far as I can see. So I think we should just keep them as top-level options